charts/argo-cd/templates/_common.tpl

```diff
@@ -46,6 +46,7 @@ helm.sh/chart: {{ include "argo-cd.chart" .context }}
 {{ include "argo-cd.selectorLabels" (dict "context" .context "component" .component "name" .name) }}
 app.kubernetes.io/managed-by: {{ .context.Release.Service }}
 app.kubernetes.io/part-of: argocd
+app.kubernetes.io/version: {{ include "argo-cd.defaultTag" . }}
 {{- with .context.Values.global.additionalLabels }}
```

Comment:
https://helm.sh/docs/chart_best_practices/labels/#standard-labels
<img width="1342" alt="image" src="https://github.com/argoproj/argo-helm/assets/30188755/3e5660fc-63d3-4de8-8f2f-2ace96ad7024">

`"argo-cd.labels"` is used to all of resources, so I think it's ok to add this label here. 🙋